src/SpectrumTransformer.cpp

```diff
@@ -357,16 +357,11 @@ bool TrackSpectrumTransformer::Process( const WindowProcessor &processor,
    bool bLoopSuccess = true;
    auto samplePos = start;
    while (bLoopSuccess && samplePos < start + len) {
-      //Get a blockSize of samples (smaller than the size of the buffer)
-      const auto blockSize = limitSampleBufferSize(
-         std::min(bufferSize, track->GetBestBlockSize(samplePos)),
-         start + len - samplePos);
```

Comment:
This change is incorrect!

Removing the use of `GetBestBlockSize` is all right, but not removing the call to `limitSampleBufferSize`, which makes sure that the last fetch does not exceed the intended range of the sequence.